magenta/scripts/convert_dir_to_note_sequences.py

```diff
@@ -168,9 +168,10 @@ def main(unused_argv):
     return
 
   input_dir = os.path.expanduser(FLAGS.input_dir)
+  output_dir = os.path.dirname(output_file)
   output_file = os.path.expanduser(FLAGS.output_file)
 
-  if not os.path.exists(os.path.dirname(output_file)):
+  if output_dir != '' and not os.path.exists(output_dir):
```

Comment:
We're trying to switch to using tf.gfile when we can for internal compatibility. It doesn't complain when the output directory already exists, so I think you could do:

```
if output_dir:
  tf.gfile.MakeDirs(output_dir)
```